src/content/plugins/internal-plugins.md

```diff
@@ -245,12 +224,6 @@ Provides AMD-style `define` and `require` to modules. Also bind `require.amd`, `
 
 Provides CommonJs-style `require` to modules.
 
-### LabeledModulesPlugin
```

Comment:
Removed in https://github.com/webpack/webpack/commit/a14e563f67292cf2bdfe28b01abd8cb5fee1e20e